packages/design-system/src/components/N8nMarkdown/Markdown.vue

```diff
@@ -162,7 +162,7 @@ export default Vue.extend({
 		};
 	},
 	methods: {
-		onClick(event) {
+		onClick(event: Event & { target?: HTMLAnchorElement }) {
```

Comment:
mouse event too?

Reply:
Tried this but `event.target` defaults to `EventTarget` so had to intersect it.

Reply:
Never mind, found the solution.